media-gfx/sane_backends/sane_backends-1.0.27.recipe

```diff
@@ -84,19 +84,12 @@ BUILD()
 	automake --add-missing
 
 	runConfigure configure \
-		--prefix=$prefix \
-		--bindir=$binDir \
-		--sbindir=$binDir \
-		--libdir=$libDir \
-		--includedir=$includeDir \
-		--datadir=$dataDir \
-		--with-docdir=$docDir \
 		--enable-pthread \
-		--disable-latex \
-		--disable-translations \
-		--enable-libusb_1_0 \
 		--enable-rpath \
 		--enable-pnm-backend \
+		--with-usb \
+		--disable-ipv6 \
+		--disable-locking \
```

Comment:
could you patch acinclude.m4#L333 instead?

Reply:
Possibly, but this is the suggested method (and well tested).